src/Element.php

```diff
@@ -195,28 +195,27 @@ public function setOption($key, $value)
      */
     public function setAttribute($key, $value)
     {
-        // Do not include the value in the list of attributes
-        if ($key === 'value') {
+        if ($key == 'value') {
+            // Do not include the value in the list of attributes
             $this->setValue($value);
-            return $this;
+        } else {
+            $this->attributes[$key] = $value;
         }
-        $this->attributes[$key] = $value;
         return $this;
```

Comment:
Please revert all of the above changes.

- We prefer strict checks (`===`), as they ensure the type as well as the value.
- We prefer returning from within conditionals, as they allow us to remove `else` conditions (and thus improve readability by reducing indentation).

The changes above are unnecessary.